manifests/s/Sass/Dart-Sass/1.83.4/Sass.Dart-Sass.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Sass.Dart-Sass
+PackageVersion: 1.83.4
+PackageLocale: en-US
+Publisher: Sass
+PublisherUrl: https://github.com/sass
+PublisherSupportUrl: https://github.com/sass/dart-sass/issues
+PackageName: Dart-Sass
+PackageUrl: https://github.com/sass/dart-sass
+License: MIT
+ShortDescription: Sass makes CSS fun again. Sass is an extension of CSS, adding nested rules, variables, mixins, selector inheritance, and more. It's translated to well-formatted, standard CSS using the command line tool or a web-framework plugin.

```

```suggestion
ShortDescription: The reference implementation of Sass, written in Dart.
Description: |-
  Sass makes CSS fun again. Sass is an extension of CSS, adding nested rules, variables, mixins,
  selector inheritance, and more. It's translated to well-formatted, standard CSS using the command
  line tool or a web-framework plugin.
Moniker: sass
```

- This "Short Description" isn't short enough. I think it'd better using the repository description.
- I'd prefer move them to the full description and manually wrap them.
- Add a moniker for it so we can keep similar behavior like _Chocolatey_ do (`winget install sass`).